app/views/shared/_ial2-alert.html.erb

```diff
@@ -0,0 +1,3 @@
+<div class="usa-error-message usa-error-message--with-icon" role="alert">
+    <%= t('sign_up.agree_to_rules_of_use_and_continue') %>
```

Comment:
Since we're reusing this text outside of sign-up, maybe we should move / rename it to something more generic? In [`forms`](https://github.com/18F/identity-idp/blob/main/config/locales/forms/en.yml) for example?